scripts/parser-tests/test262/allowlist.txt

```diff
@@ -1 +1,6 @@
-
+intl402/supportedLocalesOf-unicode-extensions-ignored.js(default)
+intl402/supportedLocalesOf-unicode-extensions-ignored.js(strict mode)
+language/import/json-invalid.js(default)
+language/import/json-invalid.js(strict mode)
+language/import/json-named-bindings.js(default)
+language/import/json-named-bindings.js(strict mode)
```

Comment:
These tests test syntax errors in an imported file, so we can't pass them with our test infrastructure.